SportHub.Business/Implementations/UserService.cs

```diff
@@ -0,0 +1,47 @@
+using Dapper;
+using Microsoft.Extensions.Configuration;
+using MySql.Data.MySqlClient;
+using SportHub.Data;
+
+namespace SportHub.Business.Implementations
+{
+    public class UserService : IUserService
+    {
+        private readonly string _connectionString;
+
+        public UserService(IConfiguration configuration)
+        {
+            _connectionString = configuration.GetConnectionString("DefaultConnection");
```

Comment:
Do not reference _connection strings directly, I sugest to create a DbConectionFactory class
```

class DbConectionFactory : IDbConectionFactory
{
    IDDConnection GetConnection()
    {
        // read conection string
        // return MySqlConnection
    }
}
```

Reply:
`using (var connection = _dbConnectionFactory.,GetCoonectio())`